docs/EAS_SECRETS_GUIDE.md

```
@@ -0,0 +1,230 @@
+    "development": {
+      "env": {
+        "EXPO_PUBLIC_GOOGLE_API_KEY_ANDROID": "@EXPO_PUBLIC_GOOGLE_API_KEY_ANDROID",
+        "EXPO_PUBLIC_GOOGLE_API_KEY_IOS": "$(EXPO_PUBLIC_GOOGLE_API_KEY_IOS)"
+      }
+    }
+  }
+}
+```
+
+EAS build logs will show:
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)